src/build/mod.rs

```diff
@@ -36,10 +36,27 @@ pub fn build(args: &BuildArgs) -> anyhow::Result<()> {
 }
 
 pub(crate) fn ensure_web_setup() -> anyhow::Result<()> {
+    // The resolved dependency graph is needed to ensure the `wasm-bindgen-cli` version matches
+    // exactly the `wasm-bindgen` version
+    let metadata = cargo::metadata::metadata()?;
+
+    let wasm_bindgen_version = metadata
+        .packages
+        .iter()
+        .find(|package| package.name == "wasm-bindgen")
```

Comment:
Cargo lets a project depend on multiple versions of the same crate. Does `wasm-bindgen` support this as well, and how should we handle it?

Reply:
I don't know if `wasm-bindgen` supports multiple versions. 

If it does It would also be possible to get the version from the `bevy_app`

Multiple `wasm-bindgen` versions:
```toml
[[package]]
name = "bevy_app"
version = "0.15.0"
source = "registry+https://github.com/rust-lang/crates.io-index"
checksum = "652574e4c10efcfa70f98036709dd5b67e5cb8d46c58087ef48c2ac6b62df9da"
dependencies = [
 "bevy_derive",
 "bevy_ecs",
 "bevy_reflect",
 "bevy_tasks",
 "bevy_utils",
 "console_error_panic_hook",
 "ctrlc",
 "derive_more",
 "downcast-rs",
 "wasm-bindgen 0.2.99",
 "web-sys",
]
```
Otherwise:

```toml
[[package]]
name = "bevy_app"
version = "0.15.0"
source = "registry+https://github.com/rust-lang/crates.io-index"
checksum = "652574e4c10efcfa70f98036709dd5b67e5cb8d46c58087ef48c2ac6b62df9da"
dependencies = [
 "bevy_derive",
 "bevy_ecs",
 "bevy_reflect",
 "bevy_tasks",
 "bevy_utils",
 "console_error_panic_hook",
 "ctrlc",
 "derive_more",
 "downcast-rs",
 "wasm-bindgen",
 "web-sys",
]
```

Not sure what's the best way to handle this, what do you think? @TimJentzsch

Reply:
This is the error message generated by `wasm-bindgen`:

> Currently the bindgen format is unstable enough that these two version must
exactly match, so it's required that these two version are kept in sync by
either updating the wasm-bindgen dependency or this binary. You should be able
to update the wasm-bindgen dependency with:
>
> ```
> cargo update -p wasm-bindgen
> ```
>
> or you can update the binary with
>
> ```
> cargo install -f wasm-bindgen-cli
> ```

From that, I assume that it doesn't make sense if an app contains two `wasm-bindgen` versions.

Even if it's somehow possible, I think it's fine if we ignore it for this first PR :)